packages/react-devtools-shared/src/devtools/views/Components/InspectedElementTree.js

```diff
@@ -73,6 +122,17 @@ export default function InspectedElementTree({
               value={value}
             />
           ))}
+        {entryToAdd && (
+          <div className={styles.AddEntry}>
+            <EditableKey overrideKeyFn={handleEntryAddName} />:
```

Comment:
Is there a way to edit an attribute after it's been added? Looks like it's only editable when it's "new". I guess it's weird to rename a prop being passed in, because a new render would re-add the previous one.

Seems like it might be nice for us to provide a way to edit (or even delete) overridden props though.

Reply:
A "reset overrides" button and/or toggle would be a nice debugging feature.

Reply:
I really like the idea from @Jessidhia ! 🚀 

I've added support for `canResetEntries`

Demo:

![reset-props](https://user-images.githubusercontent.com/23095052/64601188-c7265000-d3bc-11e9-8ae2-42fe569c20e1.gif)

Reply:
@bvaughn Regarding the ability to edit a name of an already set prop, as I mentioned in the other comment, I think we should add this support in another PR.